python_modules/libraries/dagster-celery/dagster_celery/executor_docker.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+from dagster import Executor, Field, StringSource, check, executor
+from dagster.core.definitions.executor import check_cross_process_constraints
+from dagster.core.execution.retries import Retries
+from dagster.core.host_representation.handle import IN_PROCESS_NAME
+from dagster.utils import merge_dicts
+
+from .config import DEFAULT_CONFIG, dict_wrapper
+from .defaults import broker_url, result_backend
+from .executor import CELERY_CONFIG
+
+CELERY_DOCKER_CONFIG_KEY = 'celery-docker'
+
+
+def celery_docker_config():
+    additional_config = {
+        'docker_image': Field(
+            StringSource,
+            is_required=True,
+            description='The docker image used for pipeline execution.',
+        ),
```

Comment:
could just have a whole docker section here
```
docker:
  image:
  registry:
  username:
  password:
```

Reply:
yep, now it is looking way much better, thanks